client/ayon_core/modules/base.py

```diff
@@ -1,13 +1,8 @@
 from ayon_core.addon import (
-    AYONAddon,
```

Comment:
They are here on purpose `ayon_core.modules` exists only for backwards compatibility.